tests/group_by_test.py

```diff
@@ -360,9 +359,15 @@ def test_double_same_aggregation(
         # bugged in dask https://github.com/dask/dask/issues/11612
         # and modin lol https://github.com/modin-project/modin/issues/7414
         # and cudf https://github.com/rapidsai/cudf/issues/17649
-        request.applymarker(pytest.mark.xfail)
+        request.applymarker(
+            pytest.mark.xfail(
+                reason="Backend does not support multiple aggregations with the same column"
+            )
+        )
     if "pandas" in str(constructor) and PANDAS_VERSION < (1,):
-        request.applymarker(pytest.mark.xfail)
+        pytest.skip(
+            "Pandas < 1.0.0 does not support multiple aggregations with the same column"
+        )
```

Comment:
i think this is more due to the way we've implemented it

shall we remove the reasons for tests where the condition is `backend_version < (x,)`?